File: Latihan B/java/Pc.java
// deklarasi kelas pc
public class Pc{
    // variabel private dan redefinisi kelas lain
    private Disk o_disk;
    private Processor o_proc;
    private Ram o_ram;
    private int totalprice;
    // method public dan konstruktor
    public Pc(){

    }
    public Pc(Disk disk, Processor proc, Ram ram){
        o_disk=disk;
        o_proc=proc;
        o_ram=ram;
        this.totalprice= disk.getprice() + proc.getprice() + ram.getprice();;
    }
    // get dans set semua variabel
    public Disk geto_disk(){
        return o_disk;
    }
    public void seto_disk(Disk disk){
        o_disk = disk;
    }
    public Ram geto_ram(){
        return o_ram;
    }
    public void seto_ram(Ram ram){
        o_ram = ram;
    }
    public Processor geto_proc(){
        return o_proc;
    }
    public void seto_proc(Processor proc){
        o_proc = proc;
    }
    public int gettotalprice(){
        return totalprice;
    }
    public void settotalprice(int totalprice){
        this.totalprice = totalprice;
    }
    // method untuk print
    public void printpc(){
        System.out.println("Processor Name : " + this.o_proc.getname());
        System.out.println("Processor Price : " + this.o_proc.getprice());
        System.out.println("Disk Type : " + this.o_disk.gettype());
        System.out.println("Disk Capacity : " + this.o_disk.getcapacity());
        System.out.println("Disk Price : " + this.o_disk.getprice());
        System.out.println("Ram Capacity : " + this.o_ram.getcapacity());
        System.out.println("Ram Price : " + this.o_ram.getprice());
        System.out.println("Total Price : " + this.totalprice);
    }
}

File: Latihan B/python/Pc.py
# Import
from Processor import Processor
from Disk import Disk
from Ram import Ram
# deklarasi kelas
class Pc:
    # atribut
    __oproc = Processor()
    __oram = Ram()
    __odisk = Disk()
    __totalprice = 0

    # method dan konstruktor
    def __init__(self):
        self.__oproc = Processor()
        self.__oram = Disk()
        self.__odisk = Ram()
        self.__totalprice = 0
    #  get dans set semua variabel
    def settotalprice(self, totalprice):
        self.__totalprice = totalprice
    def gettotalprice(self):
        return self.__totalprice

    def setdisk(self, Disk):
        self.__odisk = Disk
    def getdisk(self):
        return self.__odisk
    def setram(self, Ram):
        self.__oram = Ram
    def getram(self):
        return self.__oram
    def setproc(self, Proc):
        self.__oproc = Proc
    def getproc(self):
        return self.__oproc
    # method tampilkan
    def printpc(self):
        print("Processor Name : " + self.__oproc.getname())
        print("Processor Price : " + str(self.__oproc.getprice()))
        print("Disk Type : " + self.__odisk.gettype())
        print("Disk Capacity : " + self.__odisk.getcapacity())
        print("Disk Price : " + str(self.__odisk.getprice()))
        print("Ram Capacity : " + self.__oram.getcapacity())
        print("Ram Price : " + str(self.__oram.getprice()))
        print("total price: " + str(self.__totalprice))